### DISCUSSION TITLE
Implement an AI-powered code review assistant

### DISCUSSION THREAD

**Discussion by juanmaguitar** [WordPress/gutenberg#45350] [Developer Experience]
https://whatthediff.ai/
What about implementing a tool like this one in the Gutenberg repo?
It could help improve the developer experience, and Gutenberg qualifies for a [free open-source license](https://whatthediff.ai/open-source)

**Comment by sschlein** [comment]
Hey there, feel free to drop us a line at support@beyondco.de with more information and we can see what we can do 🙂

**Comment by alexstine** [comment]
@youknowriad You see this? Maybe something else to look in to along side CodeSandbox.

**Comment by mpociot** [comment]
We also added a feature yesterday that allows you to get weekly changelogs based on the repository PRs sent to you via Slack.
Here's an example: https://whatthediff.ai/repository-report/wordpress/gutenberg

If someone from the Wordpress organization signs up for a What The Diff account, I can also mark the account as sponsored and you could just give it a try on some PRs to see what it feels like.

**Comment by juanmaguitar** [comment]
Another option could be https://reviewpad.com/
They also offer a free license for Open Source projects
